shell-completion/bash/bear

```diff
@@ -0,0 +1,53 @@
+_bear() {
+  local cur prev
+  if _get_comp_words_by_ref cur prev &>/dev/null; then
+    case $prev in
+      -o|--cdb)
+        _filedir
+        COMPREPLY+=($(compgen -W 'compile_commands.json' -- "$cur"))
+        return
+        ;;
+      --use-cc|--use-c++)
+        _command_offset $COMP_CWORD
+        return
+        ;;
+      -l|--libear)
```

Comment:
this is more like developer option. (supposed to used in tests)

Reply:
I removed the "advanced options" :+1: